data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/UnorderedConsumerVerticle.java

```diff
@@ -41,10 +42,20 @@
   // This shouldn't be more than 2000, which is the default max time allowed
   // to block a verticle thread.
   private static final Duration POLL_TIMEOUT = Duration.ofMillis(1000);
+  private final int maxPollRecords;
 
-  public UnorderedConsumerVerticle(Initializer initializer,
-                                   Set<String> topics) {
+  private int inFlightRecords;
+
+  public UnorderedConsumerVerticle(final Initializer initializer,
```

Comment:
perhaps we add some javadoc on the arguments?

Reply:
Sure